sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/Samples/BindingToReceiveActions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Messaging.ServiceBus;
+using Microsoft.Azure.WebJobs.ServiceBus;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.ServiceBus.Tests.Samples
+{
+    public static class BindingToReceiveActions
+    {
+        #region Snippet:ServiceBusBindingToReceiveActions
+        [FunctionName("BindingToReceiveActions")]
+        public static async Task Run(
+            [ServiceBusTrigger("<queue_name>", Connection = "<connection_name>", IsSessionsEnabled = true)]
+            ServiceBusReceivedMessage message,
+            ServiceBusMessageActions messageActions,
+            ServiceBusReceiveActions receiveActions)
+        {
+            if (message.MessageId == "1")
```

Comment:
It may be helpful to mention that the `MessageId` has to be explicitly set by the publisher - that it isn't a field that the broker sets.

Reply:
The broker does set this if it is left blank, but in that case it would be a GUID.